packages/hydra-processor/src/migrations/IndexerStatusFields1613643992000.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class IndexerStatusFields1613643992000 implements MigrationInterface {
+  name = 'IndexerStatusFields1613643992000'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "processed_events_log" ADD COLUMN IF NOT EXISTS "indexer_head" numeric NOT NULL DEFAULT -1`
```

Comment:
Would it make sense to use `integer` over `numeric` since you already use integer for `lastScannedBlock`.